SaltSquareIO/sports-app-react-fe
# Live reloading in Docker

[CLICKUP_TICKET_ID](https://app.clickup.com/t/86c0fyer9)

## Summary of change

This pull request introduces a Docker setup that enables live reloading for the application while it is running inside a Docker container. The main changes include:
- Configured the Docker environment to support live reloading, allowing developers to see code changes immediately without restarting the container. 
- Adjusted the docker-compose.yml file to ensure the development sever correctly monitors file changes.

## QA Notes

To test this PR, follow these steps:
1. Ensure you have Docker and Docker Compose installed on your machine.
2. Run the following command to start the application in live reloading mode:

```bash
docker compose up --build
```

3. Make changes to the application code (e.g., in the src directory).
4. Verify that the changes are immediately reflected in your browser at http://localhost:3000/ without needing to restart the container.
